openedx/features/course_experience/views/course_updates.py

```diff
@@ -53,12 +53,8 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
         course_url_name = default_course_url_name(request)
         course_url = reverse(course_url_name, kwargs={'course_id': unicode(course.id)})
 
-        # Fetch all of the updates individually
-        info_module = get_course_info_section_module(request, request.user, course, 'updates')
-        ordered_updates = self.order_updates(info_module.items)
-
-        # Older implementations and a few tests store a single html object representing all the updates
-        plain_html_updates = info_module.data
+        ordered_updates = self.get_ordered_updates(request, course)
+        plain_html_updates = self.get_plain_html_updates(request, course)
```

Comment:
Nit: this probably isn't worth sweating, but is it the case that if there are plain HTML updates that we won't use the ordered updates?